src/main/java/com/cloudant/client/org/lightcouch/CouchDbDesign.java

```diff
@@ -37,23 +33,24 @@
  * DesignDocument design1 = db.design().getFromDesk("example");
  *
  * // sync with the database
- * db.design().synchronizeWithDb(design1);
+ * db.design().put(design1);
  *
  * // sync all with the database
  * db.syncDesignDocsWithDb();
  *
  * // read from the database
- * DesignDocument design2 = db.design().getFromDb("_design/example");
+ * DesignDocument design2 = db.design().get("_design/example");
  * </pre>
  *
  * @author Ahmed Yehia
- * @see {@link CouchDatabaseBase#design() db.design()} to access the API.
+ * @see {@link CouchDatabaseBase#getDesignDocumentManager() db.design()} to access the API.
  * @see DesignDocument
  * @since 0.0.2
  */
 public class CouchDbDesign {
```

Comment:
Can we just scrap this class and move whatever functionality we need up into `DesignDocumentManager`?